app/models/claim/claims_preventing_payment_finder.rb

```diff
@@ -0,0 +1,35 @@
+class Claim
+  class ClaimsPreventingPaymentFinder
```

Comment:
I think it would be useful to future-reader if we start adding Tomdoc-style comments to classes like this to give the reader some context about what it is, what it does and where it's used without having to splunk through the codebase and commit history.

Reply:
Good idea, added.